addons/website_slides_survey/models/survey_survey.py

```diff
@@ -1,11 +1,42 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models
+from odoo import models, fields, api
 
 
 class Survey(models.Model):
     _inherit = 'survey.survey'
+    slide_ids = fields.One2many('slide.slide', 'survey_id', 'Certifications')
+    slide_channel_ids = fields.One2many('slide.channel', compute='_compute_slide_channel_ids', groups='website_slides.group_website_slides_officer')
```

Comment:
We're missing a `string=` here.
Always try to add one in case we use this field in a view.

You can also add a help explaining that those are the courses linked through the e-learning app, for the same reason mentioned earlier.

Reply:
Done